GetJobber/atlantis
# build: @jobber/docz-tools package

## Motivations

We want to start using `docz` in other projects. When this happens, we want to be able to have the same theme that we have in Atlantis in other areas. This PR creates a new package for `@jobber/docz-tools` that will allow us to install the theme in other installs of docz.

We also want to make our `docz` theme more accessible and build it more with Atlantis components. There will be quite a few places in this new theme that will now use Atlantis components.

## Changes

Creates a new `@jobber/docz-tools` package that contains the `docz` theme for Atlantis.

### Added

- new `@jobber/docz-tools` package.

### Changed

- Makes theme use more Atlantis components in its build

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- pull down branch and run `npm ci`
- run `npm run clean && npm start` and see that the theme is the new theme
- run `npm run clean && npm run build && npm run serve` and see that the built code is using the new theme.

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
